packages/manager/modules/pci/src/projects/project/kubernetes/components/autoscaling/autoscaling.constants.js

```diff
@@ -0,0 +1,7 @@
+export const AUTOSCALING_LINK = {
+  DEFAULT: 'https://docs.ovh.com/gb/en/kubernetes/using-cluster-autoscaler/',
```

Comment:
Why not store this guide link here: https://github.com/ovh/manager/commits/master/packages/manager/modules/core/src/manager-core.constants.js

Reply:
The current link is specific for the Kube to scale nodePool.
I think is right location.

What do you think @JeremyDec ?

Reply:
Hmm yes it is a specific part of the documentation. IMO it annoys me to have multiple places for documentation link. But also I guess it could make the core constants heavier to store all specific links. So you can let this as is